jekyll/_cci2/server/v4.1/air-gapped-installation/phase-2-configure-object-storage.adoc

```diff
@@ -10,6 +10,7 @@ contentTags:
 :page-description: How to configure object storage through MinIO to run CircleCI server in an air-gapped environment.
 :icons: font
 :experimental:
+:circleci-agent-version: 1.0.213499-server-4.1-ecdb045d
```

Comment:
Note that this version is dependent on a patch release of 4.1: https://circleci.atlassian.net/browse/ONPREM-888